pkgs/by-name/qu/qualisys-cpp-sdk/package.nix

```diff
@@ -0,0 +1,42 @@
+{
+  cmake,
+  fetchFromGitHub,
+  fetchpatch,
+  lib,
+  stdenv,
+}:
+
+stdenv.mkDerivation {
+  pname = "qualisys-cpp-sdk";
+  version = "2024.2";
```

Comment:
How does this version relate to `refs/tags/rt_protocol_1.25`?

Reply:
This is the tag of the release: https://github.com/qualisys/qualisys_cpp_sdk/releases/tag/rt_protocol_1.25

I guess they have both internal protocol version numbers (1.25) and marketing version numbers (2024.2) synchronized in some ways.